app/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?><!--
+<!--
```

Comment:
You removed mutliple xml headers, was it on purpose?

Reply:
The XML header is optional, and AndroidStudio never adds it when creating a new file, so I decided to just remove it.

Reply:
This diff has been reverted.